BookClub/tests/views/test_meeting_list_view.py

```diff
@@ -0,0 +1,81 @@
+        'BookClub/tests/fixtures/default_meetings.json',
+        'BookCLub/tests/fixtures/default_books.json'
+    ]
+
+    def setUp(self):
+        self.club = Club.objects.get(pk=1)
+        self.meeting = Meeting.objects.get(pk=1,club=self.club)
+        self.meeting2 = Meeting.objects.get(pk=2,club=self.club)
+        self.url = reverse("meeting_list",kwargs={'club_url_name':self.club.club_url_name})
+        self.john = User.objects.get(username="johndoe")
+        self.jane = User.objects.get(username="janedoe")
+        self.jack = User.objects.get(username="jackdoe")
+        
+
+    def test_url(self):
+        url = f'/club/{self.club.club_url_name}/meetings/'
+        self.assertEqual(self.url,url)
+
+    def test_get_template_logged_in(self):
+        self.client.login(username=self.john.username, password="Password123")
+        response = self.client.get(self.url) #error here
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'club_meetings.html')
+
+    def test_redirect_when_not_logged_in(self):
+        response = self.client.get(self.url)
+        self.assertEqual(response.status_code, 302)
+
+    def test_can_see_club_name(self):
+        self.client.login(username=self.john.username, password="Password123")
+        response = self.client.get(self.url)
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'club_meetings.html')
+        self.assertContains(response, self.club.name)
+
+    def test_can_see_club_name(self):
+        self.client.login(username=self.john.username, password="Password123")
+        response = self.client.get(self.url)
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'club_meetings.html')
+        self.assertContains(response, self.club.name)
+
+    def test_can_see_a_meeting(self):
+        self.client.login(username = self.john.username, password = "Password123")
+        response = self.client.get(self.url)
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'club_meetings.html')
+        self.assertContains(response,'Book meeting 1')
+        # self.assertContains(response,'2022-02-22T19:00+00:00')
+        self.assertContains(response,'johndoe')
+
+    def test_can_see_multiple_meetings(self):
+        self.client.login(username = self.john.username, password = "Password123")
+        response = self.client.get(self.url)
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'club_meetings.html')
+
+        self.assertContains(response,'Book meeting 1')
+        self.assertContains(response,'johndoe')
+
+        self.assertContains(response,'Book meeting 2')
+        self.assertContains(response,'janedoe')
+
+    #Need to add tests involving the action views ()
+    #test contains correct info
+    #test multiple meetings
+    
```

Comment:
Are these tests done? If so, pls remove comments.

Reply:
At the moment I cannot test to see if certain users can see certain buttons. 
This is just so people know that more tests are going to be done, but just cannot be done right now.

Reply:
Could you add the buttons but not connect the URL in the template until they are completed?

Reply:
done